docs/Tutorials/Advanced/ticketing/introduction.md

```diff
@@ -2,26 +2,148 @@
+
+:::info note
+The NFT POAP is optional to include as the event organizer. You may omit it, or replace it with your own function call if you wish. In this tutorial, the POAP will be minted on the second key use. 
+:::
+
+---
+
+## Prerequisites
+
+For the advanced tutorials, you can choose to run the scripts on your own machine. To do so, you must have the following:
+
+1. [Node JS](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)  
+2. [NEAR-API-JS](https://docs.near.org/tools/near-api-js/quick-reference#install)  
+3. [Keypom JS SDK](https://github.com/keypom/keypom-js#installation)
+
+If you want to reference the finished code, that can be found [here](https://github.com/keypom/keypom-js/tree/min/ticketing-tutorial/docs-advanced-tutorials/ticket-app). To follow along and build out this ticketing app, see the steps below. 
+
+---
+
+## Creating your Project
+In this section, you're going to create your project and install the SDK to prepare for the tutorial. If you have a completed script and have installed the SDK, you can skip [forward](architecture.md).
+
+First, navigate to your desired home folder and use the `create-near-app` command to create a new project.
+``` bash
+npx create-near-app my-ticket-app
+```
+:::note
+You may realize that there is a `contract` and `frontend` folder. The `create-near-app` package allows you to build out a NEAR smart contract and quickly hook it up with a frontend. This tutorial will not involve writting any smart contracts, so the `contract` folder can be ignored. 
+:::
+
+You'll then need to create a `package.json` inside the `frontend` folder. You can accept all default values that NPM suggests.
+
+``` bash
+cd frontend && npm init
+```
+
+Then, you can install the following libraries, including the Keypom JS SDK.
+
+``` bash
+npm install keypom-js && npm install react && npm install react-dom && npm install react-router-dom &&
+npm install qrcode.react && npm install react-zxing && npm install web-vitals
+```
+
+Next, you'll want to navigate to `my-ticket-app/package.json` and make the following replacement:
+
+```json
+"scripts": {
+    ...
+    // Replace this line
+    "start": "npm run deploy && echo The app is starting! It will automatically open in your browser when ready && env-cmd -f ./neardev/dev-account.env parcel frontend/index.html --open",
+    
+    // With this
+    "start": "cd frontend && npm run start",
+    ...
+  },
+```
+
+Then, go to the `my-ticket-app/frontend/package.json` and add the following:
+``` json
+"scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html --public-url ./"
+  },
+  "devDependencies": {
+    "crypto-browserify": "^3.12.0",
+    "stream-browserify": "^3.0.0"
+  },
+```
+
+From there, create the files that will be used to build out the functionality of the ticket app. 
+
+```bash
```

Comment:
It's not clear which directory this should be run from - let's be explicit that this should be run from the frontend folder